_articles/finding-users.md

```diff
@@ -19,15 +19,15 @@ related:
 
 ## Spreading the word
 
-There's no rule that says you have to promote an open source project when you launch. There are many fulfilling reasons to work in open source that have nothing to do with popularity. If you are hoping others will find and use your open source project, however, it's time to tell everybody about your hard work!
+There's no rule that says you have to promote an open source project when you launch. There are many fulfilling reasons to work in open source that have nothing to do with popularity. Instead of hoping others will find and use your open source project, you have to _spread the word_ about your hard work!
 
 ## Figure out your message
 
-Before you start the actual work of promoting your project, you should be able to explain what it does, and why it matters.
+Before you start the actual work of promoting your project, you should be able to explain what your project does, and why your project matters.
 
-What makes your project different or interesting? Why did you create it? Answering these questions for yourself will make it easier to convince others.
+What makes your project different or interesting? Why did you create it? Answering these questions for yourself will help you communicate your project's significance.
 
-Remember that people get involved as users, and eventually contributors, because it solves a problem for them. As you think about your project's message and value, try to view it through the lens of what _they_ might want.
+Remember that people get involved as users, and eventually become contributors, because your project solves a problem for them. As you think about your project's message and value, try to view it through the lens of what _users_ might want.
 
 For example, @robb uses code examples to clearly communicate why his project, [Cartography](https://github.com/robb/Cartography), is useful:
```

Comment:
In other locations around the guide, we use GitHub usernames to refer to people with the leading `@` in place. This should be the same.

Reply:
I didn't remove the "@", GitHub is giving it a red font colour for some reason.

Reply:
Ok, thanks :grinning: